crypto/s2n_hmac.c

```diff
@@ -347,18 +373,23 @@ int s2n_hmac_copy(struct s2n_hmac_state *to, struct s2n_hmac_state *from)
  */
 int s2n_hmac_save_evp_hash_state(struct s2n_hmac_evp_backup* backup, struct s2n_hmac_state* hmac)
 {
+    ENSURE_POSIX_REF(backup);
+    PRECONDITION_POSIX(s2n_hmac_state_validate(hmac));
     backup->inner = hmac->inner.digest.high_level;
     backup->inner_just_key = hmac->inner_just_key.digest.high_level;
     backup->outer = hmac->outer.digest.high_level;
     backup->outer_just_key = hmac->outer_just_key.digest.high_level;
-    return 0;
+    return S2N_SUCCESS;
```

Comment:
Should there be a `s2n_hmac_evp_backup_validate`

Reply:
Yes, I opened issue https://github.com/awslabs/s2n/issues/2520 to track this request.